documentation/docs/how_tos/cli/running_examples.mdx

```diff
@@ -0,0 +1,23 @@
+---
+description: History behind the development of the Stronghold Engine library that reflects upon previous revisions and the lessons learnt.
+image: /img/logo/Stronghold_icon.png
+keywords:
+- examples
+- cli
+- client crate
+- running examples
+---
+# Running Examples
+
+## Command Line Interface(CLI)
+
+You should run the examples from within the [client crate](https://github.com/iotaledger/stronghold.rs/tree/dev/client).
+The examples shown in this doc were executed on Linux but should work on the other supported platforms. Make sure to
+adapt paths according to your operating system default.
+
+For example, Windows(TM) requires backslashes `\` as a delimiter for a path. This could pose a problem, as
```

Comment:
Is the TM really needed? 😅 
```suggestion
For example, Windows requires backslashes `\` as a delimiter for a path. This could pose a problem, as
```